### ISSUE TITLE
Worldsmith cannot run without rhoyne.cloudapp.net:8010 being up

### ISSUE THREAD

**Issue by SinZ163** [RoyAwesome/WorldSmith#86]
Crash report: (for consistancy, **Build136**)
```
Please report this crash to https://github.com/RoyAwesome/worldsmith/issues 
Please include your project and the build that crashed (located in the Version.txt file)

System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 191.239.34.229:8010
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)
   at System.Net.WebClient.DownloadString(Uri address)
   at UpdaterLib.Updater.QuickDownloadJson(String url) in c:\buildbot\slave\build\build\UpdaterLib\UpdaterLib.cs:line 54
   at UpdaterLib.Updater.GetChannelManifest(UpdateChannel channel, String url) in c:\buildbot\slave\build\build\UpdaterLib\UpdaterLib.cs:line 24
   at UpdaterLib.Updater.CheckForUpdate(UpdateChannel channel, DateTime LastUpdate, String url) in c:\buildbot\slave\build\build\UpdaterLib\UpdaterLib.cs:line 43
   at WorldSmith.CheckForUpdate.Check(Boolean Silently)
   at WorldSmith.Program.Main()
```

**Comment by RoyAwesome** [comment]
Fixed the build server not being up, I'll fix the code to better gracefully handle this

**Comment by RoyAwesome** [comment]
fixed with a0b1f45b4e6f40f52e87b24aef973d271d07d66b
